src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslX509CertificateReader.cs

```diff
@@ -245,10 +245,7 @@ public RSA GetRSAPrivateKey()
                 return null;
             }
 
-            using (SafeRsaHandle rsaHandle = Interop.libcrypto.EVP_PKEY_get1_RSA(_privateKey))
```

Comment:
Seems reasonable.  When this code was written there wasn't an SafeEvpPKeyHandle constructor :)